v2/src/log_hresult.c

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <inttypes.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "windows.h"
+#include "psapi.h"
+
+#include "c_logging/format_message_no_newline.h"
+#include "c_logging/log_hresult.h"
+
+#define MESSAGE_BUFFER_SIZE 512
+#define N_MAX_MODULES 10
```

Comment:
>N_MAX_MODULES [](http://example.com/codeflow?start=8&length=13)

"It is a good idea to specify a large array of HMODULE values"...

from [https://learn.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-enumprocessmodules](https://learn.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-enumprocessmodules)

:)

Reply:
Made it ten fold